controllers/storagecluster/generate.go

```diff
@@ -171,3 +171,8 @@ func generateCephReplicatedSpec(initData *ocsv1.StorageCluster, poolType string)
 func generateStorageQuotaName(storageClassName, quotaName string) string {
 	return fmt.Sprintf("%s-%s", storageClassName, quotaName)
 }
+
+// generateNameForCephSubvolumeGroup function generates a name for CephFilesystemSubVolumeGroup
+func generateNameForCephSubvolumeGroup(filesystemName string) string {
```

Comment:
generateNameForCephSubvolumeGroup -> generateNameForDefaultCephSubvolumeGroup

Reply:
The function name should be generic, I don't think we should add default in the function name itself